src/neo/Ledger/Blockchain.cs

```diff
@@ -62,21 +62,17 @@ private class UnverifiedBlocksList { public LinkedList<Block> Blocks = new Linke
         private static readonly object lockObj = new object();
```

Comment:
Maybe this method here can be removed from Blockchain.cs as well: https://github.com/neo-project/neo/blob/6793253ef238921e86bc52a156ad5c83c64f6ae1/src/neo/Ledger/Blockchain.cs#L132-L135

Bot to Ledger, but in order to clean it, this is a little bit apart for its general view as it is right now.

Reply:
Done.